ceph.spec.in

```diff
@@ -1244,6 +1244,13 @@ Group:          System/Monitoring
 %description prometheus-alerts
 This package provides Ceph default alerts for Prometheus.
 
+%package mib
+Summary:        MIB for SNMP alerts
+BuildArch:      noarch
+Group:          System/Monitoring
```

Comment:
might want to guard this with `%if 0%{?suse_version}`?

Reply:
@bigjust hi Justin, sorry for the confusion, i meant to suggest that we should guard
```
Group: System/Monitoring
```
with  `%if 0%{?suse_version}` as the `Group` is a SUSE only thing. see https://en.opensuse.org/openSUSE:Package_group_guidelines . while fedora stopped using it, see https://fedoraproject.org/wiki/RPMGroups

Reply:
Ah I see! Thanks for the guidance :), I've updated the commit